tests/lib/rules/eol-last.js

```diff
@@ -26,10 +27,22 @@ ruleTester.run("eol-last", rule, {
         "var a = 123;\n\n",
         "var a = 123;\n   \n",
 
-        "\r\n",
-        "var a = 123;\r\n",
-        "var a = 123;\r\n\r\n",
-        "var a = 123;\r\n   \r\n"
+        { code: "", options: ["unix"] },
```

Comment:
Can we group together the tests that are for the deprecated options with a comment labeling them as such in both the valid and invalid tests (maybe at the end of each grouping)? Will make it much easier for the next person who might be removing the options in a future major version.